libs/shared/common-services/src/validation.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ValidationService {
+  public validationPassed$: BehaviorSubject<boolean> = new BehaviorSubject(true);
```

Comment:
public is not necessary, moreover you could make it constant using the `readonly` attribute
```suggestion
  readonly validationPassed$: BehaviorSubject<boolean> = new BehaviorSubject(true);
```